.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+defaults: &defaults
+  working_directory: /home/ubuntu/
+  steps:
+    - checkout
+    - run:
+        name: Prepare environment
+        command: pip install -e .
+    - run:
+        name: Run tests
+        command: PYTHONPATH=. pytest
+
+setup:
+
+version: 2
+jobs:
+  python27:
+    <<: *defaults
+    docker:
+      - image: python:2.7
```

Comment:
Let's drop it? If anyone still wants to use it on py2 a year after py2 is officially EOL, they're free to use an older commit.